ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -357,6 +323,20 @@ instance SupportsTwoPhaseValidation ConwayEra where
         ) -> True
     _ -> False
 
+instance SupportsTwoPhaseValidation DijkstraEra where
+  -- TODO(geo2a): I do not understand this code. Do I need to?
+  isIncorrectClaimedFlag _ = \case
+    SL.ConwayUtxowFailure
+      ( Conway.UtxoFailure
+          ( Conway.UtxosFailure
+              ( Conway.ValidationTagMismatch
+                  (Alonzo.IsValid _claimedFlag)
+                  _validationErrs
+                )
+            )
+        ) -> True
+    _ -> False
```

Comment:
This code is fine; it should actually live in Ledger, cf https://github.com/IntersectMBO/cardano-ledger/issues/4167 :sweat_smile:

Reply:
Maybe replace that `TODO` with a link to https://github.com/IntersectMBO/cardano-ledger/issues/4167